app/objects/c_mapper.py

```diff
@@ -0,0 +1,18 @@
+from app.utility.base_object import BaseObject
+
+
+class Mapper(BaseObject):
+
+    @property
+    def unique(self):
+        return '%s%s%s%s' % (self.source, self.edge, self.target, self.json_key)

```

Comment:
not every mapper is going to have a json key. Here I would think about allowing any number of arbitrary properties to be added.

Reply:
I imagined additional properties could be added specifically to the Mapper object and other places, but that's more work than needed. I'll change the object to have a 'misc' dict or some other name.